Orange/preprocess/impute.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, variable, value=0):
 
     def transform(self, c):
         if issparse(c):     # sparse does not have unknown values
```

Comment:
Can you please remove the comment `# sparse does not have unknown values` that doesn't apply any more?